platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/hash/VirtualHasher.java

```diff
@@ -62,25 +63,14 @@ public final class VirtualHasher<K extends VirtualKey, V extends VirtualValue> {
      */
     private static final Logger logger = LogManager.getLogger(VirtualHasher.class);
 
-    /**
-     * Since {@code com.swirlds.platform.Browser} populates settings, and it is loaded before any
-     * application classes that might instantiate a data source, the {@link ConfigurationHolder}
-     * holder will have been configured by the time this static initializer runs.
-     */
-    private static final VirtualMapConfig CONFIG = ConfigurationHolder.getConfigData(VirtualMapConfig.class);
-
-    /**
-     * The number of threads to use when hashing. Can either be supplied by a system property, or
-     * will compute a default based on "percentHashThreads".
-     */
-    private static final int HASHING_THREAD_COUNT = CONFIG.getNumHashThreads();
-
     /**
      * This thread-local gets a HashBuilder that can be used for hashing on a per-thread basis.
      */
     private static final ThreadLocal<HashBuilder> HASH_BUILDER_THREAD_LOCAL =
             ThreadLocal.withInitial(() -> new HashBuilder(Cryptography.DEFAULT_DIGEST_TYPE));
 
+    private final VirtualMapConfig vmConfig = ConfigurationHolder.getConfigData(VirtualMapConfig.class);
```

Comment:
Should this still be here?

Reply:
It is no longer static, so the original problem is not applicable here. However, I see your point. There is no need to have a field for this config, it can be queried as needed in the `hash()` method